pkgs/development/libraries/tbb/default.nix

```diff
@@ -25,6 +25,10 @@ with stdenv.lib; stdenv.mkDerivation rec {
     rm $out/include/index.html
   '';
 
+  postFixup = ''
+    substituteAll ${./FindTBB.cmake} $out/FindTBB.cmake
+  '';
+
```

Comment:
How is this supposed to work? If my memory serves correctly, our CMake setup hooks do not search `$out` but actually search some path like `$out/share/cmake` for all `buildInputs` in order to find `.cmake` files. You'd have to have some other workaround for this anyway, wouldn't you?